lib/utils/utils.go

```diff
@@ -191,14 +191,15 @@ var (
 	ErrFindProjectRoot = errors.New("cannot find project root")
 )
 
-// GetProjectRootPath finds the root of the project where `go.mod` is
+// GetProjectRootPath finds the root of the project where directory `cmd`
+// and subdirectories `gossamer, testcases` are
 // and returns it as an absolute path.
 func GetProjectRootPath() (rootPath string, err error) {
 	_, fullpath, _, _ := runtime.Caller(0)
 	rootPath = path.Dir(fullpath)
 
-	const directoryToFind = "chain"
-	const subPathsToFind = "dev,gssmr,kusama,polkadot"
+	const directoryToFind = "cmd"
+	const subPathsToFind = "gossamer,testcases"
```

Comment:
supernit let's just have `gossamer`, since `testcases` might be renamed/disappear/be-moved in the future

```suggestion
	const subPathsToFind = "gossamer"
```